src/plugin/hooks.ts

```
@@ -0,0 +1,119 @@
+/**
+ * React hooks for plugin system integration
+ */
+
+import { useState, useEffect } from 'react';
+import pluginManager from './manager';
+import { ExtensionPointType, IDrawerExtension, IRouteExtension } from './types';
+
+export function usePluginDrawerItems(
+  userPermissions: string[] = [],
+  isAdmin: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)